specification/maps/data-plane/Render/preview/1.0/render.json

```diff
@@ -351,7 +352,8 @@
     },
     "/map/tile/{format}": {
       "get": {
-        "description": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\n\nFetches map tiles in vector or raster format typically to be integrated into a new map control or SDK. By default, Azure uses vector map tiles for its web map control (see [Zoom Levels and Tile Grid](https://docs.microsoft.com/en-us/azure/location-based-services/zoom-levels-and-tile-grid))\n\n**Note**: Weather tiles are only available via [Get Map Tile V2 API](https://aka.ms/AzureMapsWeatherTiles). We recommend to start to use the new [Get Map Tile V2 API](https://aka.ms/GetMapTileV2).",
+        "summary": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\n\nFetches map tiles in vector or raster format typically to be integrated into a new map control or SDK. By default, Azure uses vector map tiles for its web map control (see [Zoom Levels and Tile Grid](https://docs.microsoft.com/en-us/azure/location-based-services/zoom-levels-and-tile-grid))\n\n**Note**: Weather tiles are only available via [Get Map Tile V2 API](https://aka.ms/AzureMapsWeatherTiles). We recommend to start to use the new [Get Map Tile V2 API](https://aka.ms/GetMapTileV2).",
+        "description": "> [!NOTE]\n>\n> **Azure Maps Render v1 service retirement**\n>\n> The Azure Maps [Render v1](/rest/api/maps/render-v1) service is now deprecated and will be retired on 9/17/26. To avoid service disruptions, all calls to Render v1 API will need to be updated to use  [Render v2](/rest/api/maps/render-v2) API by 9/17/26.\n\n**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\n\nFetches map tiles in vector or raster format typically to be integrated into a new map control or SDK. By default, Azure uses vector map tiles for its web map control (see [Zoom Levels and Tile Grid](https://docs.microsoft.com/en-us/azure/location-based-services/zoom-levels-and-tile-grid))\n\n**Note**: Weather tiles are only available via [Get Map Tile V2 API](https://aka.ms/AzureMapsWeatherTiles). We recommend to start to use the new [Get Map Tile V2 API](https://aka.ms/GetMapTileV2).",
```

Comment:
It seems that `[Render v1](/rest/api/maps/render-v1)` is a broken link. Do you know what it's supposed to link to instead?

Reply:
The `[Render v2](/rest/api/maps/render-v2)` link seems to work, but I suggest using the actual web link (https://learn.microsoft.com/en-us/rest/api/maps/render-v2) instead as I'm not sure the file reference will work once deployed.

Reply:
@ismael-ms Just fixed the broken link for Render V1 and change both links for V1 and V2 to actual web link